examples/decoding/plot_receptive_field.py

```diff
@@ -130,9 +130,9 @@
 ax.set(title="Mean prediction score", xlabel="Channel", ylabel="Score ($r$)")
 mne.viz.tight_layout()
 
+###############################################################################
 # Investigate model coefficients
-# ------------------------------
-#
+# ==============================
```

Comment:
I think these should be single `-----` not double `======` but double check that this is true.

Reply:
Actually there seems to be a nice little feature in reSt markup whereby : 
> The following are all valid section title adornment characters: `= -  : . ' " ~ ^ _ * + #`
> Rather than imposing a fixed number and order of section title adornment styles, the order enforced will be the order as encountered.The first style encountered will be an outermost title (like HTML H1), the second style will be a subtitle, the third will be a subsubtitle, and so on.
http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#sections

So here `---` should render as my Big section title, and `===` as my secondary title. On my machine at least it seems to render well.

Reply:
I'm referring to the fact that here we are using double lines for headers:

https://github.com/nbara/mne-python/blob/98606a4d496081b165505a476d8612d1b67e0866/examples/decoding/plot_receptive_field.py#L135

and here (and elsewhere) we are using single lines:

https://github.com/nbara/mne-python/blob/98606a4d496081b165505a476d8612d1b67e0866/examples/decoding/plot_receptive_field.py#L86

though now that I look at it, it seems like this may be intentional, in which case ignore this comment. Just making sure that it is, in fact, intentional.

Reply:
Yeah, fully intentional. The idea is to separate decoding and encoding analysis better. 

Single line will render as a heading, and double lines as a subheading. Looks fabulous ;)

Reply:
https://5786-1301584-gh.circle-artifacts.com/0/tmp/circle-artifacts.Z9QLSD8/html/auto_examples/decoding/plot_receptive_field.html

Reply:
Thanks @larsoner, I was about to ask if you could get the link to the example. 

What do you both think ?

Reply:
@nbara in the future you can do it, too, go to the CircleCI link, login with GitHub, go to artifacts.

I'll wait until @choldgraf is happy then take a final look and merge :)